FoursquareTopTests/VenueDetailViewControllerDriver.swift

```diff
@@ -29,4 +31,63 @@ class VenueDetailViewControllerDriver {
             tester.waitForAbsenceOfViewWithAccessibilityLabel(categoryName)
         }
     }
-}
\ No newline at end of file
+    
+    func expectMenuButton(toBeVisible visible: Bool) {
+        if visible {
+            testCase.waitForViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        } else {
+            testCase.waitForAbsenceOfViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        }
+    }
+    
+    func expectAddressLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.address, withAccessibilityLabel: tr(.VenueDetailAddressAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectStatusLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.status, withAccessibilityLabel: tr(.VenueDetailStatusAccessibilityLabel), toBeVisible: visible)
+      
+    }
+    
+    func expectRatingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.formattedRating, withAccessibilityLabel: tr(.VenueDetailRatingAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectPricingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool)
+    {
```

Comment:
swiftlint should be configured only allow 1TBS indentation style (I am being a total smart ass after checking this wikipedia article! https://en.wikipedia.org/wiki/Indent_style#1TBS) xD